server/src/constants.ts

```diff
@@ -0,0 +1 @@
+export const SUPPORTED_VERSIONS = ['2.0.1-1.18.2']
```

Comment:
Is it `2.0.1-1.18.2` or `2.0.1-1.18.2-fabric`?

Reply:
fug, you're right